### ISSUE TITLE
Evaluation Mode doesn't work.

### ISSUE THREAD

**Issue by Leanna97** [huawei-noah/Efficient-AI-Backbones#202]
Thanks for your great work. I train my 'vig_ti_224_gelu' model and the accuracy reaches 78.22%. These results are from the summary.csv generated during training. When I try to resume the saved checkpoint using '--resume', it works well. However, when I tried to evaluate the saved model using '--evaluate', the results become 1.04% and the loss is nan. Could anyone help me to solve this problem?

summary.csv:
![image](https://user-images.githubusercontent.com/87753077/235120730-b5b228aa-c698-42a5-9da8-a86582c2a85b.png)

'--evaluate':
![image](https://user-images.githubusercontent.com/87753077/235122534-8c0188a7-7509-476e-a12f-44df3dddd470.png)

**Comment by iamhankai** [comment]
I think it's the version issue. Please refer to https://github.com/huawei-noah/Efficient-AI-Backbones/issues/219#issuecomment-1649928890

**Comment by FreeZ3e** [comment]
Same problem, I tried torchvision==0.8.2, torch==1.7.1, timm==0.3.2 and CUDA11.0. But didn't work in evaluate mode.

**Comment by xxrrnn** [comment]
我通过改变imagenet的文件结构解决了问题。我的imagenet结构中，val里面是很多jpeg，而train里面是很多文件夹。根据#219老哥的colab，应该是val里面很多文件夹，train里面1个文件夹。因此只需要将二者的名称交换，就能够输出正确的结果。

**Comment by FreeZ3e** [comment]
I solved this problem by using "--resume"  to load pretrained model instead of "--pretrain_path".
